presto-docs/src/main/sphinx/release/release-320.rst

```diff
@@ -0,0 +1,66 @@
+===========
+Release 320
+===========
+
+General Changes
+---------------
+
+* Fix incorrect parameter binding order when parameters appear inside a
+  ``WITH`` clause. (:issue:`1191`)
+* Fix planning failure for certain queries involving a mix of ``OUTER``
+  and ``CROSS`` joins. (:issue:`1589`)
+* Improve performance of queries containing complex predicates. (:issue:`1515`)
+* Avoid unnecessary evaluation of redundant filters. (:issue:`1516`)
+* Improve performance of certain window functions when using bounded window
+  frames. E.g., ``(ROWS BETWEEN ... PRECEDING AND ... FOLLOWING)``. (:issue:`464`)
+* Add :doc:`/connector/kinesis`. (:issue:`476`)
+* Add :func:`geometry_from_hadoop_shape`. (:issue:`1593`)
+* Add :func:`at_timezone`. (:issue:`1612`)
+* Add :func:`with_timezone`. (:issue:`1612`)
+
+JDBC Driver Changes
+-------------------
+
+* Warnings for queries are now available exclusively via ``Statement``, not
+  ``ResultSet``. (:issue:`1640`)
```

Comment:
@electrum maybe mention that this is a "more correct" this way, essentially a bug fix.

Reply:
How about
```
* Only report warnings on ``Statement``, not ``ResultSet``, as warnings
  are not associated with reads of the ``ResultSet``. (:issue:`1640`)